web/.eslintrc

```diff
@@ -0,0 +1,10 @@
+{
+  "parserOptions": {
+    "ecmaVersion": 2021,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true // Allows for the parsing of JSX
+    }
+  },
+  "extends": ["plugin:@typescript-eslint/recommended"]
```

Comment:
Might also want to extend eslint:recommended and prettier.

I think you might also want the following, but I can't remember why. :)
```
  parser: '@typescript-eslint/parser',
  parserOptions: {project: './tsconfig.json'},
```